snippets/sdk/python/configurations/archive_file.py

```diff
@@ -0,0 +1,61 @@
+#! /usr/local/env python3
+
+import base64
+import os
+
+from azure.identity import DefaultAzureCredential
+from azure.mgmt.nginx import NginxManagementClient
+
+
+def main():
+    SUBSCRIPTION_ID = os.environ.get("AZURE_SUBSCRIPTION_ID", None)
+    GROUP_NAME = "myResourceGroup"
+    DEPLOYMENT_NAME = "myDeployment"
+    CONFIGURATION_NAME = "default"  # configuration must be named default
+
+    # Create clients
+    # For other authentication approaches, please see: https://pypi.org/project/azure-identity/
+    nginx_client = NginxManagementClient(
+        credential=DefaultAzureCredential(), subscription_id=SUBSCRIPTION_ID
+    )
+
+    # Check that you have created your deployment already
+    # For snippets to create a deployment navigate to ../deployments/create_or_update.py
+    nginx_client.deployments.get(GROUP_NAME, DEPLOYMENT_NAME)
+
+    # Read in our configuration archive
+    cwd = os.getcwd()
+    filepath = os.path.join(os.path.dirname(__file__), "configs", "config.tar.gz")
+    file = open(filepath, "rb")
+    content = file.read()
+    b64_content = base64.b64encode(content).decode("ascii")
+
+    # Create multi file configuration from the archive in the deployment
+    configuration = nginx_client.configurations.begin_create_or_update(
+        GROUP_NAME,
+        DEPLOYMENT_NAME,
+        CONFIGURATION_NAME,
+        {
+            "properties": {
+                "rootFile": "/etc/nginx/nginx.conf",
+                "package": {"data": b64_content},
+            }
+        },
+    ).result()
+    print("Created a configuration:\n{}".format(configuration))
+
+    # Get configuration
+    deployment = nginx_client.configurations.get(
+        GROUP_NAME, DEPLOYMENT_NAME, CONFIGURATION_NAME
+    )
+    print("Get configuration:\n{}".format(deployment))
+
+    # Delete our configuration from the deployment
+    nginx_client.configurations.begin_delete(
+        GROUP_NAME, DEPLOYMENT_NAME, CONFIGURATION_NAME
+    )
```

Comment:
do we need a `result()` like the others?

```suggestion
    ).result()
```